src/ufixed.jl

```diff
@@ -35,14 +31,7 @@ const uf14 = UFixedConstructor{UInt16,14}()
 const uf16 = UFixedConstructor{UInt16,16}()
 
 zero{T,f}(::Type{UFixed{T,f}}) = UFixed{T,f}(zero(T),0)
-for uf in UF
-    TT = rawtype(uf)
-    f = nbitsfrac(uf)
-    T = UFixed{TT,f}
-    @eval begin
-        one(::Type{$T}) = $T($(2^f-1),0)
-    end
-end
+one{T<:UFixed}(::Type{T}) = T((2^nbitsfrac(T)-1),0)
```

Comment:
good catch.  thanks.  it was 4x slower.   fixed for 0.4 and 0.5 with a generated function.  that seems a simpler solution to me than having to switch on the version.